src/apps/ocioview/ocioview/utils.py

```diff
@@ -31,18 +34,18 @@ def __enter__(self) -> None:
             obj.blockSignals(True)
 
     def __exit__(
-        self, exc_type: type, exc_val: Exception, exc_tb: TracebackType
+            self, exc_type: type, exc_val: Exception, exc_tb: TracebackType
```

Comment:
Ah!, this should not have been committed, I might have staged the entire file instead of the lines. I will undo that. I'm living in the future, `ruff format`! It has a slightly different formatting than `black`.